djcelery_email/tasks.py

```diff
@@ -19,7 +19,11 @@
 
 
 @shared_task(**TASK_CONFIG)
-def send_emails(messages, backend_kwargs):
+def send_emails(messages, backend_kwargs={}, **kwargs):
```

Comment:
Never use a mutable type as an argument default. It can result in strange bugs if the argument is mutated and then the subsequent calls get the modified version.